.github/workflows/reusable-uki-test.yaml

```diff
@@ -1,203 +0,0 @@
-            exit 1
-          else
-            echo "No critical vulnerabilities found in the image."
-          fi
-      - name: Build uki ISO 🔧
-        run: |
-          mkdir build
-          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
-          -v $PWD/tests/assets/keys:/keys \
-          -v $PWD/tests/assets/sysext/:/overlay \
-          -v $PWD/build/:/output \
-          quay.io/kairos/auroraboot:v0.10.0 --debug build-uki --output-dir /output/ --output-type iso \
-          --public-keys /keys \
-          --tpm-pcr-private-key /keys/tpm2-pcr-private.pem \
-          --sb-key /keys/db.key --sb-cert /keys/db.pem \
-          --single-efi-cmdline="testentry: nothing" \
-          --overlay-iso /overlay \
-          docker:${{ env.IMAGE_NAME }}
-      - name: Build uki upgrade image 🔧
-        if: ${{ inputs.test_label != 'boot-assessment' }}
-        run: |
-          echo "Building upgrade image for ${{ inputs.test_label }}"
-          docker build -t upgrade-image - <<EOF
-          FROM ${{ env.IMAGE_NAME }}
-          RUN echo "Current release info: \$(cat /etc/kairos-release)"
-          RUN sed -i 's/^KAIROS_VERSION=.*/KAIROS_VERSION="mynewversion"/' /etc/kairos-release
-          RUN echo "New release info: \$(cat /etc/kairos-release)"
-          EOF
-
-          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
-            -v $PWD/build:/result \
-            -v $PWD/tests/assets/keys:/keys \
-            quay.io/kairos/auroraboot:v0.10.0 --debug build-uki \
-            --output-dir /result --output-type container \
-            --public-keys /keys \
-            --tpm-pcr-private-key /keys/tpm2-pcr-private.pem \
-            --sb-key /keys/db.key --sb-cert /keys/db.pem \
-            docker:upgrade-image
-
-          image=$(docker load -i build/*.tar | grep "Loaded image" | awk -F: '{ st = index($0,":");print substr($0,st+1)}' | sed 's/+/-/g' | xargs)
-          echo "Image: $image"
-          rand=$(head /dev/urandom | tr -dc a-z0-9 | head -c 7)
-          temp_image=quay.io/kairos/ci-temp-images:$(echo $image | sed "s/:latest/-${rand}/")
-          docker build --label quay.expires-after="6h" --tag $temp_image - <<EOF
-          FROM $image
-          EOF
-          docker push $temp_image
-          echo $temp_image > build/upgrade_image_name
-      - name: Run tests 🔧
-        env:
-          USE_QEMU: true
-          KVM: true
-          MEMORY: 6000
-          CPUS: 4
-          FIRMWARE: /usr/share/OVMF/OVMF_CODE.fd
-          EMULATE_TPM: true
-          UKI_TEST: true
-        run: |
-          export ISO=$(ls $PWD/build/kairos-*-uki.iso)
-          export UPGRADE_IMAGE=$(cat "${PWD}/build/upgrade_image_name")
-          export EXPECTED_NEW_VERSION="mynewversion"
-          export EXPECTED_SINGLE_ENTRY="testentry"
-          cp tests/go.* .
-          go run github.com/onsi/ginkgo/v2/ginkgo -v --label-filter "${{ inputs.test_label }}" --fail-fast -r ./tests/
```

Comment:
so we don't run these anymore?

Reply:
@jimmykarily we do, here: https://github.com/kairos-io/kairos/blob/master/.github/workflows/uki.yaml#L158

Reply:
it's already on master